docs/api-reference/actions/actions.md

```diff
@@ -340,7 +340,7 @@ is listened by all subreducers,
 ### addDataToMap
 
 Add data to kepler.gl reducer, prepare map with preset configuration if config is passed.
-Kepler.gl provides a handy set of utils to parse data from different format to the `data` object required in dataset. You rarely need to manually format the data obejct.
```

Comment:
This files are auto-generated by script. change made in here will be override next time, you should change in the source file: JsDoc of that actions.js where `addDataToMap` is exported from

Reply:
Ad didnt realize that. Made the updates https://github.com/keplergl/kepler.gl/pull/914